Source/i_video.c

```diff
@@ -1056,20 +1056,57 @@ static void I_RestoreDiskBackground(void)
   disk_to_draw = 0;
 }
 
+static const float gammalevels[18] =
+{
+    // Darker
+    0.50f, 0.55f, 0.60f, 0.65f, 0.70f, 0.75f, 0.80f, 0.85f, 0.90f,
+
+    // No gamma correction
+    1.0f,
+
+    // Lighter
+    1.125f, 1.25f, 1.375f, 1.5f, 1.625f, 1.75f, 1.875f, 2.0f,
+};
+
+byte gamma2table[18][256];
+
+static void I_InitGamma2Table(void)
+{
+  int i, j;
+
+  for (i = 0; i < 18; ++i)
+    for (j = 0; j < 256; ++j)
+    {
+      gamma2table[i][j] = (byte)(pow(j / 255.0, 1.0 / gammalevels[i]) * 255.0 + 0.5);
```

Comment:
This results in `255.5` for `j == 255`. Is this properly rounded or should we use e.g. `floor()`?

Reply:
It's converted to a byte (unsigned character), so I think it's ok.